src/components/RevokeContainer/RevokeContainer.tsx

```diff
@@ -25,19 +26,24 @@ export const RevokeContainer: FunctionComponent = () => {
 
       const validateDocument = async () => {
         const network = config?.network || "";
-        let isDocumentValid = true;
+        let isDocumentValid = false;
+        let errors = [] as string[];
 
         if (network !== "local") {
           const verify = verificationBuilder(openAttestationVerifiers, { network: network });
           const fragments = await verify(revokeDocuments[0]);
-          isDocumentValid = isValid(fragments);
+          errors = errorMessageHandling(fragments);
         }
 
-        if (!isDocumentValid) {
-          setErrorMessage("Document cannot be read. Please check that you have a valid document");
+        if (errors.length > 0) {
+          setErrorMessage(CONSTANTS.MESSAGES[errors[0]].failureMessage);
```

Comment:
why only `[0]` ? show all pushed errors?

Reply:
ok updated it to show all errors